client-src/elements/chromedash-drawer.ts

```diff
@@ -261,6 +261,12 @@ export class ChromedashDrawer extends LitElement {
     const myFeaturesMenu = this.renderMyFeaturesMenu();
     const adminMenu = this.renderAdminMenu();
 
+    const year = new Date().getFullYear();
```

Comment:
Just a heads-up that on Jan 1, this will cause a lot of the playwright screenshot tests to break when the link changes from `Shipping 2024` to `Shipping 2025`.

Reply:
Good point. I'll follow up with a PR to elide that.